doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.md

```diff
@@ -88,7 +70,74 @@ $ kubectl logs raycluster-autoscaler-head-xxxxx -c autoscaler | tail -n 20
 # ...
 ```
 
-## Ray Autoscaler vs. other autoscalers
+(kuberay-autoscaling-config)=
+## KubeRay Config Parameters
+
+There are two steps to enabling Ray autoscaling in the KubeRay `RayCluster` custom resource (CR) config:
+
+1. Set `enableInTreeAutoscaling:true`. The KubeRay operator will then automatically configure an autoscaling sidecar container
+for the Ray head pod. The autoscaler container collects resource metrics from the Ray cluster
+and automatically adjusts the `replicas` field of each `workerGroupSpec` as needed to fulfill
+the requirements of your Ray application.
+
+2. Set the fields `minReplicas` and `maxReplicas` to constrain the number of `replicas` of an autoscaling
+`workerGroup`. When deploying an autoscaling cluster, one typically sets `replicas` and `minReplicas`
+to the same value.
+The Ray autoscaler will then take over and modify the `replicas` field as pods are added to or removed from the cluster.
+
+For an example, check out the [config file](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.autoscaler.yaml) that we used in the above quickstart guide.
```

Comment:
at some point, we should link to something static (as opposed to a master branch on github)